addons/rating/models/mail_thread.py

```diff
@@ -23,7 +23,45 @@ def unlink(self):
         return result
 
     def _get_message_create_valid_field_names(self):
-        return super()._get_message_create_valid_field_names() | {'rating_ids'}
+        return super()._get_message_create_valid_field_names() | {"rating_id", "rating_value"}
```

Comment:
Note: there is a lot of code for `rating_id` while it seems almost never used.
Main call is livechat `_post_feedback_message` which could be refactored to pass `rating_value` most of the time.
And changing rating from message doesn't seem very useful. It should probably be prevented, but each model kind of does its own:
- there is a check on livechat route, but it is not enforced in models
- there is a different check for website slides
- I've not checked others

Overall rating should be fully cleaned, just doing the minimal here to keep features intact.